### ISSUE TITLE
hhvm-dev missing glog

### ISSUE THREAD

**Issue by simonwelsh** [hhvm/packaging#79]
After managing to compile gcc-4.9 on Debian Wheezy, was unable to compile hhvm-pgsql as the hhvm-dev package did not include the glog headers, but `base-includes.h` ends up including it through folly.

**Comment by jbboehr** [comment]
Having this issue too, with Ubuntu precise:

```
$ make
Scanning dependencies of target handlebars
[100%] Building CXX object CMakeFiles/handlebars.dir/handlebars.cpp.o
In file included from /usr/include/hphp/third-party/folly/Conv.h:30:0,
                 from /usr/include/hphp/third-party/folly/Format.h:32,
                 from /usr/include/hphp/runtime/base/datatype.h:24,
                 from /usr/include/hphp/runtime/base/typed-value.h:24,
                 from /usr/include/hphp/runtime/base/complex-types.h:20,
                 from /usr/include/hphp/runtime/ext/extension.h:21,
                 from /home/travis/build/jbboehr/hhvm-handlebars/handlebars.cpp:5:
/usr/include/hphp/third-party/folly/Range.h:28:26: fatal error: glog/logging.h: No such file or directory
 #include <glog/logging.h>
                          ^
compilation terminated.
make[2]: *** [CMakeFiles/handlebars.dir/handlebars.cpp.o] Error 1
make[1]: *** [CMakeFiles/handlebars.dir/all] Error 2
make: *** [all] Error 2
```

TravisCI is using it for C++ builds.

**Comment by fredemmott** [comment]
For the most part, this class of problems is fixed in the new WIP nightly system (see #128 (comment) ) - built on a clean system, runtime depends are inferred from build-depends; if something's wrong with the package name/version, we won't have the required build dependencies, so it'll fail to build.

This isn't fixed for wheezy, and that's a wontfix.

**Comment by jbboehr** [comment]
At the time, precise was the only supported OS on TravisCI and pre-built HHVM packages weren't available yet, but I've long since stopped testing my libraries against HHVM, so no big deal.
